FreeRTOS/FreeRTOS-Kernel
# Remove the sample smp configuration folder

Description
-----------
Remove the sample smp configuration folder. Define the following configurations in cmake instead
* configNUMBER_OF_CORES = 2
* configUSE_PASSIVE_IDLE_HOOK=0

```cmake
if (DEFINED FREERTOS_SMP_EXAMPLE AND FREERTOS_SMP_EXAMPLE STREQUAL "1")
    message(STATUS "Build FreeRTOS SMP example")
    # Adding the following configurations to build SMP template port
    add_compile_options( -DconfigNUMBER_OF_CORES=2 -DconfigUSE_PASSIVE_IDLE_HOOK=0 )
endif()
```

Test Steps
-----------
The SMP template port build is included in CI action already.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
